uint16[6] angle # dynamixel: 12bit[ 0 ~ 4096]
